src/ray/gcs/gcs_server/gcs_kv_manager.cc

```diff
@@ -141,6 +141,23 @@ void GcsInternalKVManager::HandleInternalKVKeys(
   }
 }
 
+void GcsInternalKVManager::HandleGetInternalConfig(
```

Comment:
This doesn't need to be an separate rpc, we can just make it a key-value entry with a pre-defined key and just call `Get` to fetch it?